src/routes/package-show-route/package-show-route.test.js

```diff
@@ -249,6 +249,25 @@ const resource = {
   attributes: {
     name: 'Title name 1',
     isSelected: false,
+    managedCoverages: [{
+      'beginCoverage': '2005-01-01',
```

Comment:
Could you please remove  '  (from 'beginCoverage': '2005-01-01', to beginCoverage: '2005-01-01',) for curent line and below?